src/lib/core/components/visualizations/implementations/LineplotVisualization.tsx

```diff
@@ -2078,10 +2076,12 @@ function getRequestParams(
   const viewport =
```

Comment:
After we hear back from the UX crew, I think we can remove the viewport from the request in all circumstances.

Reply:
@bobular I didn't test the case without viewport, but if we completely remove it, then I think that binWidth won't automatically change for no binning case (vizConfig.useBinning: false)?

Reply:
I'm not sure what you mean by "binWidth won't automatically change for no binning case"?

When not binning, do we care about binWidth?

Reply:
@bobular If I remember correctly, for the case of vizConfig.useBinning: false, binWidth is automatically changed/adjusted if X-axis custom range is changed (for a certain range only though)?

Reply:
@bobular viewport is removed

Reply:
Sounds like my imagined semantic zooming on lineplot x axis!